evaluation/data/modis/sorc/regrid_modis_to_gaussian.f90

```diff
@@ -0,0 +1,723 @@
+
+error = nf90_get_var(ncid, varid(1), modis_bsa_vis)
+ call netcdf_err(error, 'reading modis_bsa_vis variable')
+
+!  read white sky modis alebdos
+
+error = nf90_inq_varid(ncid, 'Albedo_WSA_shortwave', varid(1))
+ call netcdf_err(error, 'inquire modis_wsa_sw variable' )
+
+error = nf90_get_var(ncid, varid(1), modis_wsa_sw)
+ call netcdf_err(error, 'reading modis_wsa_sw variable' )
+
+error = nf90_inq_varid(ncid, 'Albedo_WSA_NIR', varid(1))
+ call netcdf_err(error, 'inquire modis_wsa_nir variable' )
+
+error = nf90_get_var(ncid, varid(1), modis_wsa_nir)
+ call netcdf_err(error, 'reading modis_wsa_nir variable')
+
+error = nf90_inq_varid(ncid, 'Albedo_WSA_VIS', varid(1))
+ call netcdf_err(error, 'inquire modis_wsa_vis variable' )
+
+error = nf90_get_var(ncid, varid(1), modis_wsa_vis)
+ call netcdf_err(error, 'reading modis_wsa_vis variable')
+    
+error = nf90_close(ncid)
+ call netcdf_err(error, 'closing file: '//trim(modis_filename) )
+
+allocate(data_regrid(idestinations, jdestinations))
+allocate(percent_good_data(idestinations, jdestinations))
+
+!====================================
+! write remapped albedo
+
+output_filename = trim(regrid_outpath)//'/'//trim(fv3_grid)//'/MCD43C3.A'//trim(daystring)//'.061.'//trim(fv3_grid)//'.nc'
+
+error = nf90_create(output_filename, ior(nf90_netcdf4,nf90_classic_model), ncid)
+ call netcdf_err(error, 'creating file='//trim(output_filename) )
+
+! --- define longitude, latitude,  and time
+
+error = nf90_def_dim(ncid, 'lat', jdestinations, dim_id_j)
+ call netcdf_err(error, 'defining lat dimension' )
+
+error = nf90_def_dim(ncid, 'lon', idestinations, dim_id_i)
+ call netcdf_err(error, 'defining lon dimension' )
+
+error = nf90_def_dim(ncid, 'time', NF90_UNLIMITED, dim_time)
+  call netcdf_err(error, 'defining time dimension' )
+
+error = nf90_def_var(ncid, 'time', nf90_double, dim_time, id_time)
+  call netcdf_err(error, 'defining time' )
+error = nf90_put_att(ncid, id_time, "long_name", "time")
+  call netcdf_err(error, 'defining time long name' )
+error = nf90_put_att(ncid, id_time, "units", "seconds since "//since_date)
+  call netcdf_err(error, 'defining time units' )
+
+!--- define longitude
+error = nf90_def_var(ncid, 'lon', nf90_float, (/dim_id_i/), id_lon)
+  call netcdf_err(error, 'defining lon' )
+error = nf90_put_att(ncid, id_lon, "long_name", "longitude")
+  call netcdf_err(error, 'defining lon long name' )
+
+!--- define latitude
+error = nf90_def_var(ncid, 'lat', nf90_float, (/dim_id_j/), id_lat)
+  call netcdf_err(error, 'defining lat' )
+error = nf90_put_att(ncid, id_lat, "long_name", "latitude")
+  call netcdf_err(error, 'defining lat long name' )
+
+!======================================
+! white-sky albedo
+! shortwave
+
+error = nf90_def_var(ncid, 'albedo_wsa_shortwave', nf90_short, (/dim_id_i,dim_id_j, dim_time/), varid(1))
+ call netcdf_err(error, 'defining albedo_wsa_shortwave' )
+
+error = nf90_put_att(ncid, varid(1), "_FillValue", nodata_int)
```

Comment:
similar to above, there should be a description attribute on all these variables so that it is clear what they are in the netcdf file

Reply:
The same as above